src/drivers/src/motors_def_cf2.c

```diff
@@ -581,3 +606,14 @@ const MotorPerifDef* motorMapBoltBrushless[NBR_OF_MOTORS] =
   &BOLT_M4_BL
 };
 
+/**
+ * Brushless motors mapped to the Bolt 1.1 PWM outputs.
+ */
+const MotorPerifDef* motorMapBolt11Brushless[NBR_OF_MOTORS] =
+{
+  &BOLT_M1_BL,
+  &BOLT_M2_BL,
+  &BOLT_M3_BL,
+  &BOLT11_M4_BL
```

Comment:
Why is this one called BOLT11_M4_BL while the other ones are called BOLT_M#_BL?

Reply:
We could decouple it from the platform and call it e.g. PA1_TIM2_CH2_BL and do the same with all of them.